docs/about/Removed.rst

```diff
@@ -192,3 +192,9 @@ Replaced with a GUI version: `gui/unit-syndromes`.
 warn-stuck-trees
 ================
 The corresponding DF :bug:`9252` was fixed in DF 0.44.01.
+
+.. gui/dig:
```

Comment:
Ha, the fact that the PR didn't fail tipped me off that there was an issue here. Missing an underscore

Reply:
Also please keep in alphabetical order